cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -1164,7 +1164,7 @@ Usage: cardano-cli shelley transaction txid
 
   Print a transaction identifier.
 
-Usage: cardano-cli shelley transaction view [--output-format STRING]
+Usage: cardano-cli shelley transaction view [--json | --yaml]
```

Comment:
I'm wondering if we should prefix those formats with `output` or similar e.g. `--output-json`. It would be better to logically group them together in case we would like to add new output formats e.g. cbor, bson, messagepack, xml...

Reply:
I think I prefer the shorter flags, but no strong opinion. @Jimbo4350> what do you think?

Reply:
OK so went for `--output-json` and `--output-yaml` after seeing @Jimbo4350's similar opinion below :+1:

Reply:
Yep that's fine :+1: